Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -922,4 +917,75 @@ protected override void ProcessRecord()
             }
         }
     }
+
+    [Cmdlet(VerbsCommon.Copy, "GcsObject", DefaultParameterSetName = ParameterSetNames.ByObject)]
+    [OutputType(typeof(Object))]
+    public class CopyGcsObject : GcsCmdlet
```

Comment:
Know what I could go for? Some XML doc comments. A delicious "summary" and "description" tag would be great right about now.